lago/plugins/vm.py

```diff
@@ -179,35 +183,52 @@ def interactive_console(self):
         """
         return self.vm.interactive_ssh()
 
-    def extract_paths(self, paths):
+    def extract_paths(self, paths, ignore_nopath):
         """
         Extract the given paths from the domain
 
         Args:
             paths(list of str): paths to extract
+            ignore_nopath(boolean): if True will ignore none existing paths.
+
+        Returns:
+            None
+
+        Raises:
+            :exc:`~lago.plugins.vm.ExtractPathNoPathError`: if a none existing
+                path was found on the VM, and ``ignore_nopath`` is True.
+            :exc:`~lago.plugins.vm.ExtractPathError`: on all other failures.
         """
-        if self.vm.alive() and self.vm.ssh_reachable():
-            self._extract_paths_scp(paths=paths)
-        elif self.vm.alive():
-            raise ExtractPathError(
-                'Unable to extract logs from alive but unreachable host %s. '
-                'Try stopping it first' % self.vm.name()
-            )
+        if self.vm.alive() and self.vm.ssh_reachable(
```

Comment:
If it's alive but not reachable via SSH, don't we won't to fallback to the old method of fetching files?